lib/kernel/doc/src/file.xml

```diff
@@ -401,6 +401,52 @@ f.txt:  {person, "kalle", 25}.
         </taglist>
       </desc>
     </func>
+    <func>
+      <name name="del_dir" arity="2" since="OTP 23.0"/>
+      <fsummary>Delete a directory.</fsummary>
+      <desc>
+        <p>Deletes directory <c><anno>Dir</anno></c> as specified by
+          <c><anno>Opts</anno></c>, which contains zero or more
+          of the following options:</p>
+        <taglist>
+          <tag><c>recursive</c></tag>
+          <item>
+            <p>The contents of the directory is recursively deleted
+             before the directory itself is.</p>
+          </item>
+          <tag><c>force</c></tag>
+          <item>
+            <p>Ignores errors when accessing or deleting a file or
+             directory.</p>
+          </item>
+          <tag><c>keeptop</c></tag>
+          <item>
+            <p>The top-level directory is not deleted.</p>
```

Comment:
Should we clarify that if this option is given and `recursive` isn't, then the operation is a no-op? :)

Reply:
Why deny people of that easter-egg? :-)  But yes, I could clarify that if needed.